### DISCUSSION TITLE
https://nextapps-de.github.io/spotlight/

### DISCUSSION THREAD

**Discussion by Adinomis** [community/community#128837] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Greate one! Really simple, if you need simple one!
The only file that I am using is 'spotlight.bundle.js'
But, small tipo or (re)quest: I couldn't find how to make description disapearing slower (when walking through of course). Many ppl told me that time of visibility is too short.
Maybe you can help with this one? Which line in spotlight.bundle or any else...

Thanks again,

Regards, Dag

**Comment by ebndev** [comment]
Hi @Adinomis, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from specific project repo, is that right?

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

If you don't mind, would you mind letting us know how you ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your Discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board. 

Any insight would be helpful, thanks!

**Reply by Adinomis** [reply]
This is about nothing. My question were so simple... so simple...
